src/neo/SmartContract/NefFile.cs

```diff
@@ -85,7 +92,8 @@ public class NefFile : ISerializable
         public void Serialize(BinaryWriter writer)
         {
             SerializeHeader(writer);
-            writer.Write((short)0);
+            writer.WriteVarString(Source);
```

Comment:
Use the reserved byte in order to be compatible?

Reply:
We have to use var string for the compatibility.

Reply:
With var string, all old nef files have a `source` field with an empty string value.